src/common/options/mds.yaml.in

```diff
@@ -1105,6 +1105,18 @@ options:
   services:
   - mds
   with_legacy: true
+# Max number of slow ops to track
+- name: mds_op_history_slow_op_size
+  type: uint
+  level: advanced
+  default: 20
+  with_legacy: true
+# track the op if over this threshold
+- name: mds_op_history_slow_op_threshold
+  type: float
+  level: advanced
+  default: 10
+  with_legacy: true
```

Comment:
we don't add new options with `with_legacy`. Please rewrite without.

Reply:
I have remove the "with_legacy" and git file, thanks!